tests/Makefile

```diff
@@ -1,32 +1,29 @@
 # Tests that should be run each time
-RUNTESTS := ordering encoding metadata tls reconnect
+RUNTESTS := ordering metadata tls 
```

Comment:
Should we re-add the `reconnect` test? Maybe it can be just tested directly in the root package, without any code gen.?